rollup.config.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Shared config used by individual packages' Rollup configs
```

Comment:
It will eventually have everything, at least for the bundles. Once our npm builds also move to rollup, there may well be `rollup.config.bundle.js` and `rollup.config.npm.js`, but we can cross that bridge when we come to it.